Analytical/Classes/Provider/FirebaseProvider.swift

```diff
@@ -24,268 +24,257 @@ public enum FirebaseProperties : String {
     case travelClass
 }
 
-public class FirebaseProvider : BaseProvider<Firebase.Analytics>, AnalyticalProvider {
-    
+public class FirebaseProvider: BaseProvider<Firebase.Analytics>, AnalyticalProvider {
     public static let GoogleAppId = "GoogleAppIdKey"
     public static let BundleId = "BundleIdKey"
+    public static let APIKey = "APIKey"
+    public static let ProjectID = "ProjectID"
     public static let GCMSenderId = "GCMSenderID"
     
-    public func setup(with properties: Properties?) {
+    public func setup(with properties: Analytical.Properties?) {
```

Comment:
You should be inside `Analytical` module, so no need to reference it explicitly.